### ISSUE TITLE
Is the Directive compatible with fullcalendar 3?

### ISSUE THREAD

**Issue by enjoyvinh** [angular-ui/ui-calendar#498]
I want use FullCalendar 3 with ui-calendar that is it Ok?
Please help me, thanks.

**Comment by rdeslonde-eichleay** [comment]
+1

**Comment by theblang** [comment]
Yeah, we couldn't use this wrapper since it is still on `fullcalendar 2.x`. @peterver @zachdunn , do you have any idea if there are major hurdles involved with updating to `3.x`? I may be interested in attempting to do so.

Also, @rdeslonde-eichleay , you probably want to stick to reactions for +1's as using comments is annoying for everyone subscribed to the issue (which in this case isn't many yet, but often times is a lot). It is quite frustrating to constantly get notifications for every issue you have been subscribed to that just contain `+1`.

**Comment by enjoyvinh** [comment]
@mattblang thanks you so much.
I hope to have the new update supports `fullcalendar 3.x` as soon as possible.

**Comment by oostmeijer** [comment]
+1

**Comment by shakisha** [comment]
no update on this?
